README.md

```diff
@@ -111,42 +67,33 @@ this. Add an entry for each of your ILP kits in your /etc/hosts file, like so:
 To send from Kit 0 to Kit 3, log into `ilp-kit0:2010` as `admin`. Send a payment
 to `admin@ilp-kit3:5010` in the sending UI.
 
-### Custom Header File
-
-If you need any additional containers in the docker-compose (eg. a bitcoin
-node, an extra five-bells-ledger, etc.), you can set an alternate header file.
-This will replace the start of the docker-compose (before the ILP Kit
-containers are added) with a file of your choice.
-
-```sh
-$ node generate-docker-compose.js --ledger example.json --header header.yml > docker-compose.yml
-$ docker-compose build
-$ docker-compose up -d
-$ docker-compose logs -f
-```
 
-## Configuration Format Specification
+## Configuration Format
 
 The JSON file used for the ILP Demo maker contains an array of objects in the
-below format.
+below format. See [`example.json`](./example.json) for a sample config file.
 
 ```js
-{
-  "prefix": "example.ledgerA.", // (1)
-  "currency": "USD", // (2)
-  "plugin": "ilp-plugin-A", // (3)
-  "store": true, // [optional] (4)
-  "rpcUri": true, // [optional] (5)
-  "rpcUris": true, // [optional] (6)
-  "left_account": "example.ledgerA.alice", // (7)
-  "left_config": { // (8)
-    /* ... */
-  },
-  "right_account": "example.ledgerA.bob",
-  "right_config": {
-    /* ... */
+[
```

Comment:
I don't think this makes it much more confusing and it's better to have the full format in there. Before you also had a big example in the readme but this was an attempt to kind of combine the two